src/master/flags.cpp

```diff
@@ -460,6 +460,18 @@ mesos::internal::master::Flags::Flags()
       "load an alternate allocator module using `--modules`.",
       DEFAULT_ALLOCATOR);
 
+  add(&Flags::hierarchical_recovery_factor,
+      "hierarchical_recovery_factor",
+      "Ratio of minimal re-registred agent before sending\n"
+      "offers after a leader re-election",
+      DEFAULT_AGENT_RECOVERY_FACTOR);
```

Comment:
`DEFAULT_ALLOCATOR_AGENT_RECOVERY_FACTOR`? 
Looks like the already existing `Flags` code follows the convention that the default has the same name as the flag, prefixed with `DEFAULT_`?